docs/rules/arrow-parens.md

```diff
@@ -49,13 +49,14 @@ a => {}
 
 ## Options
 
-The rule takes one option, a string, which could be either `"always"` or `"as-needed"`. The default is `"always"`.
+This rule has a string option:
```

Comment:
My preference would be to note at the top level that this also has a second object option and then go into more detail into each option. That way the reader can know at a glance that they should keep going down the page if they're interested in all the options.